auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -461,21 +577,274 @@ export default defineComponent({
       }
     }
 
-    watch(() => props.selectedColumns, (newCol: string[]) => {
-      getHeaders(newCol)
-    })
+    const enableNewActionsMenu = (): boolean => {
+      return launchdarklyServices.getFlag(LDFlags.EnableNewActionsMenu) || false
+    }
+
+    const isModernizedEntity = (item: Business): boolean => {
+      const entityType = getEntityType(item)
+      const supportedEntityFlags = launchdarklyServices.getFlag(LDFlags.IaSupportedEntities)?.split(' ') || [] // move to somewhere factory?
+      console.log(supportedEntityFlags)
+      return supportedEntityFlags.includes(entityType)
+    }
+
+    const isColinEntity = (item: Business): boolean => { // not include ones modernized (feature flag)
+      const entityType = getEntityType(item)
+      // talk to Sev to see if there's better way to check colin
```

Comment:
It might be better to refactor the logic so that "COLIN" is the last choice in goToRegister().

(Right now, you don't check "isOtherEntities" in goToRegister().)

I would ask Janis or Mihai which is the better last/fallback choice: COLIN or forms.

Reply:
This conversation is resolved by a change to `goToRegister` below.

Reply:
I still think this isn't the right way to check for a "COLIN" entity. 

I think (but please confirm if this makes sense), that if the entity is not modernized (according to FF), and not a Society, and not an Other, then it's a COLIN. See comment below re: goToRegister().

Reply:
This was done in a recent commit. Thanks! Resolving.